pkgs/development/misc/juce/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+
+# Native build inputs
+, cmake
+, pkg-config
+, makeWrapper
+
+# Dependencies
+, alsa-lib
+, freetype
+, curl
+, libglvnd
+, webkitgtk
+, pcre
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Use the finalAttrs pattern: https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/253561/commits/a8da2f90d78bc8ce15f5964980a2536e0b14ed16